### PR TITLE
Develop to Master

### PR DESCRIPTION
**Pull request by chiragjn**
Minor changes -> mostly C/I fixes and datastore create related changes
Major changes -> New v2/text API for detecting multiple text entities on multiple texts at once!

### CODE REVIEW

File: datastore/elastic_search/create.py [hunk 1]
```diff
@@ -1,9 +1,16 @@
+import logging
```
Comment (haptik-deployment) [inline datastore/elastic_search/create.py:1]: W1618 [34mimport missing `from __future__ import absolute_import`[0m

**Comment by haptik-deployment** [comment]
UNIT TESTS HAVE PASSED... Good To Merge

**Comment by haptik-deployment** [comment]
UNIT TESTS HAVE PASSED... Good To Merge

**Comment by haptik-deployment** [comment]
UNIT TESTS HAVE PASSED... Good To Merge

**Review by ankur09011 — APPROVED** [review]

**Pull request merged**